lib/stacker/stack.rb

```diff
@@ -141,8 +160,77 @@ def update options = {}
       end
     end
 
+    def describe_change_set
+      retries = 6
+      changes = []
+      while changes.empty?
+        resp = region.client.describe_change_set(
+          change_set_name: change_set,
+          stack_name: name
+        )
+        changes = resp.changes
+        if changes.empty?
+          raise CannotDescribeChangeSet.new 'Empty change set' if retries == 0
+          retries -= 1
+          sleep 1
```

Comment:
Change set creation is non-blocking, so it might not be ready for reading immediately. The request can succeed, but the change list will be empty. I haven't investigated enough to be sure there's no cleaner way to wait.